std/json.d

```diff
@@ -359,6 +359,90 @@ struct JSONValue
         return type == JSONType.null_;
     }
 
+    /***
+     * Generic type value getter
+     * A convenience getter that returns `JSONValue` as the specified D type.
+     * Note: only arithmetic, `bool`, `string`, `JSONValue` and `JSONValue[]` types are accepted
+     * Throws: `JSONException` if `T` is not a valid type or conversion is impossible
+     */
+    @property inout(T) get(T)() inout const pure @safe
+    {
+        static if (is(T : string))
+        {
+            return str;
+        }
```

Comment:
Nit, single-statement blocks waste vertical space.

Reply:
If you don't mind, I will leave this as is for style consistency across this file. Many functions (eg `assign`, `opBinary`, `opAssign`, `toValue`) have multiple one-line ifs implemented in similar way. I'm okay with changing this if you insist though.